test/unit/seek-to-live.test.js

```diff
@@ -2,96 +2,88 @@
 import TestHelpers from './test-helpers.js';
 import sinon from 'sinon';
 import computedStyle from '../../src/js/utils/computed-style.js';
+import { createTimeRange } from '../../src/js/utils/time-ranges.js';
+
```

Comment:
I re-wrote most of these tests, they still test the same things but there was a ton of code duplicated between the modules and most of the tests were a variation of the same thing.